sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -294,6 +303,35 @@ export interface FileFlushOptions extends CommonOptions {
     retainUncommittedData?: boolean;
 }
 
+// @public (undocumented)
+export interface FileParallelUploadOptions extends CommonOptions {
+    // (undocumented)
+    abortSignal?: AbortSignalLike;
+    chunkSize?: number;
+    // (undocumented)
+    close?: boolean;
```

Comment:
what does `close` mean?

Reply:
It's the same `close` as in `FileFlushOptions`.

Reply:
Xiaoning might suggesting to add comments in both place.

---
In reply to: [388116604](https://github.com/Azure/azure-sdk-for-js/pull/7659#discussion_r388116604) [](ancestors = 388116604)